### PR TITLE
Eric's review of README.md

### PR DESCRIPTION
**Pull request by erickoeck**
Mostly some formatting and highlighting changes to match Intel style.

### CODE REVIEW

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Latency Control
 This FPGA tutorial demonstrates how to set latency constraints to pipes and LSUs accesses.
 
-***Documentation***:  The [DPC++ FPGA Code Samples Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of DPC++ for FPGA. <br>
-The [oneAPI DPC++ FPGA Optimization Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) is the reference manual for targeting FPGAs through DPC++. <br>
-The [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general resource for target-independent DPC++ programming.
+#### Additional Documentation:
```
Comment (yuguen) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:4]: This is a format that is used across all code samples and we cannot change it without going through the process of asking the code samples committee about it

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md [hunk 2]
```diff
@@ -105,23 +112,23 @@ Third party program Licenses can be found here: [third-party-programs.txt](https
 ## Building the `latency_control` Tutorial
 > **Note**: If you have not already done so, set up your CLI
 > environment by sourcing  the `setvars` script located in
-> the root of your oneAPI installation.
+> the root of your oneAPI installation as follows:
```
Comment (yuguen) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:115]: We cannot change this section either

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md [hunk 3]
```diff
@@ -105,23 +112,23 @@ Third party program Licenses can be found here: [third-party-programs.txt](https
...
+If running a sample in the Intel DevCloud, remember that you must specify the type of compute node and whether to run in batch or interactive mode. Compiles to FPGA are supported only on `fpga_compile` nodes. Executing programs on FPGA hardware is only supported on `fpga_runtime` nodes of the appropriate type, such as `fpga_runtime:arria10` or `fpga_runtime:stratix10`.  Neither compiling nor executing programs on FPGA hardware are supported on the login nodes. For more information, see the *Intel® oneAPI Base Toolkit Get Started Guide* ([https://devcloud.intel.com/oneapi/documentation/base-toolkit/](https://devcloud.intel.com/oneapi/documentation/base-toolkit/)).
```
Comment (yuguen) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:129]: We cannot change this section

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md [hunk 4]
```diff
@@ -135,9 +142,9 @@ The basic steps to build and run a sample using VS Code include:
  - Run the sample in the VS Code terminal using the instructions below.
 
 To learn more about the extensions and how to configure the oneAPI environment, see
-[Using Visual Studio Code with Intel® oneAPI Toolkits](https://software.intel.com/content/www/us/en/develop/documentation/using-vs-code-with-intel-oneapi/top.html).
+[*Using Visual Studio Code with Intel® oneAPI Toolkits*](https://software.intel.com/content/www/us/en/develop/documentation/using-vs-code-with-intel-oneapi/top.html).
```
Comment (yuguen) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:145]: We cannot change this section

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md [hunk 5]
```diff
@@ -215,29 +222,31 @@ After learning how to use the extensions for Intel oneAPI Toolkits, return to th
 *Note:* The Intel® PAC with Intel Arria® 10 GX FPGA and Intel® FPGA PAC D5005
 (with Intel Stratix® 10 SX) do not support Windows*. Compiling to FPGA hardware
 on Windows* requires a third-party or custom Board Support Package (BSP) with
-Windows* support.<br> *Note:* If you encounter any issues with long paths when
+Windows* support.
```
Comment (yuguen) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:225]: We cannot change this section

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md [hunk 6]
```diff
@@ -215,29 +222,31 @@ After learning how to use the extensions for Intel oneAPI Toolkits, return to th
...
+
+ *Tip:* If you encounter any issues with long paths when
 compiling under Windows*, you may have to create your ‘build’ directory in a
 shorter path, for example c:\samples\build. You can then run cmake from that
 directory, and provide cmake with the full path to your sample directory.
 
 ### Troubleshooting
-If an error occurs, you can get more details by running `make` with
+If an error occurs, get more details by running `make` with
```
Comment (yuguen) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:233]: We cannot change this section

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md [hunk 7]
```diff
@@ -215,29 +222,31 @@ After learning how to use the extensions for Intel oneAPI Toolkits, return to th
...
 For more comprehensive troubleshooting, use the Diagnostics Utility for
-Intel® oneAPI Toolkits, which provides system checks to find missing
+Intel® oneAPI Toolkits that provides system checks to find missing
 dependencies and permissions errors.
 [Learn more](https://software.intel.com/content/www/us/en/develop/documentation/diagnostic-utility-user-guide/top.html).
 
  ### In Third-Party Integrated Development Environments (IDEs)
 
-You can compile and run this tutorial in the Eclipse* IDE (in Linux*) and the Visual Studio* IDE (in Windows*). For instructions, refer to the following link: [Intel® oneAPI DPC++ FPGA Workflows on Third-Party IDEs](https://www.intel.com/content/www/us/en/developer/articles/technical/intel-oneapi-dpcpp-fpga-workflow-on-ide.html)
+You can compile and run this tutorial in the Eclipse* IDE (in Linux*) and the Visual Studio* IDE (in Windows*). For instructions, refer to the following link: [*Intel® oneAPI DPC++ FPGA Workflows on Third-Party IDEs*](https://www.intel.com/content/www/us/en/developer/articles/technical/intel-oneapi-dpcpp-fpga-workflow-on-ide.html)
```
Comment (yuguen) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:243]: We cannot change this section

**Review by yuguen — COMMENTED** [review]
Thanks Erick for having a look at these READMEs
Unfortunately there are multiple sections that we cannot change as they have been curated by the code samples committee and they are replicated in all samples.
We would have to request the committee to review a proposed change a do a bulk edit
I've marked the sections you edited that we can't modify
Shuo will implement the other changes
I'll get I'll get in touch with you for the next README review so that you don't waste your time on sections we can't change
Sorry about that :(
And thanks for your input, that's really appreciated!

**Comment by erickoeck** [comment]
Just curious…was there any tech comms representation on the code samples committee when developing the README templates?

**Comment by yuguen** [comment]
I honestly don't know - but there are at least two "README persons" in the committee
Maybe there are tech comms but I'm not sure

**Comment by erickoeck** [comment]
Do you know their names?

**Comment by yuguen** [comment]
I usually ask Tom Lenth to review the READMEs, but I know there is Isha Ghosh as well

**Comment by erickoeck** [comment]
Hmmm…I don't know them, but workers.intel.com says they're usability engineers, not technical writers/documentarians.

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md (hunk 3 above)
Comment (erickoeck) [reply DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:129]: Just FYI, this is a grammar error. The "only" is in the grammatically incorrect position.

Comment (yuguen) [reply DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:129]: English not being my native language I can only trust you!
I'll suggest that you get invited to the committee when discussing README changes, I'll let you know how this goes

**Pull request closed without merging**